### DISCUSSION TITLE
Release of Django 3.2 support?

### DISCUSSION THREAD

**Discussion by karisal-anders** [django-import-export/django-import-export#1310] [General]
Hello!
According to 8e6c0e6992fbf9c7b10e0aff491af27ec1cd9600 there's Django 3.2 support in the master.
Would it be possible to make an official release with these changes and push it to PyPi for general use?
This would be greatly appreciated, thank you!

Kari Salminen

**Comment by matthewhegarty** [comment]
Hi - we are looking into producing the next release as soon as we have availability

**Reply by karisal-anders** [reply]
Thanks for the info! How has the availability been lately? Would you dare give a guesstimate for a possible release time or are there too many unknowns to make such an estimate?

**Reply by matthewhegarty** [reply]
hopefully in the next week or two - but I cannot guarantee it because it is dependent on the availability of others.

**Reply by karisal-anders** [reply]
Alright, thank you!

**Reply by karisal-anders** [reply]
Hello!
How's the possibility of the release looking currently? I know this is an open source project and you work on it voluntarily so there's no obligation involved.

**Reply by matthewhegarty** [reply]
Hi Kari - it is planned and being prepped but we are just waiting for team availability, so it should be by the end of the month.

**Reply by matthewhegarty** [reply]
Release 2.6.0 now available on [pypi.org](https://pypi.org/project/django-import-export/)

**Reply by karisal-anders** [reply]
> Release 2.6.0 now available on [pypi.org](https://pypi.org/project/django-import-export/)

Thank you! 👍
